From 6717d02ccb105a5828f231b663f524482bb7992a Mon Sep 17 00:00:00 2001
From: Xu Yang <xu.yang_2@nxp.com>
Date: Tue, 28 Jun 2022 16:05:38 +0800
Subject: [PATCH 172/274] MLK-25966 arm64: dts: imx93-11x11-evk-i3c: ptn5110
 using shared interrupt line with pcal6524

This workaround will use a shared interrupt line with pcal6524 to
support level triggered tcpci driver.

Signed-off-by: Xu Yang <xu.yang_2@nxp.com>
Reviewed-by: Clark Wang <xiaoning.wang@nxp.com>
---
 arch/arm64/boot/dts/freescale/imx93-11x11-evk-i3c.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx93-11x11-evk-i3c.dts b/arch/arm64/boot/dts/freescale/imx93-11x11-evk-i3c.dts
index 775241406..5c77a5475 100644
--- a/arch/arm64/boot/dts/freescale/imx93-11x11-evk-i3c.dts
+++ b/arch/arm64/boot/dts/freescale/imx93-11x11-evk-i3c.dts
@@ -66,8 +66,8 @@ lsm6dso_i3c: imu@6a,208006c0000 {
 	ptn5110: tcpc@50 {
 		compatible = "nxp,ptn5110";
 		reg = <0x50 0x00 0x50>;
-		interrupt-parent = <&pcal6524>;
-		interrupts = <1 IRQ_TYPE_EDGE_FALLING>;
+		interrupt-parent = <&gpio3>;
+		interrupts = <27 IRQ_TYPE_LEVEL_LOW>;
 		status = "okay";
 
 		port {
@@ -93,8 +93,8 @@ typec1_con: connector {
 	ptn5110_2: tcpc@51 {
 		compatible = "nxp,ptn5110";
 		reg = <0x51 0x00 0x50>;
-		interrupt-parent = <&pcal6524>;
-		interrupts = <9 IRQ_TYPE_EDGE_FALLING>;
+		interrupt-parent = <&gpio3>;
+		interrupts = <27 IRQ_TYPE_LEVEL_LOW>;
 		status = "okay";
 
 		port {
-- 
2.25.1

